frontend/packages/dev-console/src/components/pipelineruns/logs/pipelineRunLogSnippet.ts

```diff
@@ -4,26 +4,11 @@ import {
   PLRTaskRunData,
   PLRTaskRunStep,
 } from '../../../utils/pipeline-augment';
+import { CombinedErrorDetails } from './log-snippet-types';
 import { pipelineRunStatus } from '../../../utils/pipeline-filter-reducer';
+import { taskRunSnippetMessage } from './log-snippet-utils';
 
-type ErrorDetails = {
-  title: string;
-};
-
-type ErrorDetailsWithLogName = ErrorDetails & {
-  containerName: string;
-  podName: string;
-};
-type ErrorDetailsWithStaticLog = ErrorDetails & {
-  staticMessage: string;
-};
-
-export type PipelineRunErrorDetails = ErrorDetailsWithLogName | ErrorDetailsWithStaticLog;
-
-const joinConditions = (conditions: Condition[]) =>
-  conditions.map((condition) => condition.message).join('\n') || 'Unknown failure condition';
-
-export const getLogSnippet = (pipelineRun: PipelineRun): PipelineRunErrorDetails => {
+export const getLogSnippet = (pipelineRun: PipelineRun): CombinedErrorDetails => {
```

Comment:
Would appear I failed to write tests for this method. Would you mind doing that as part of this PR?

Reply:
This PR is dragging for a long time. Maybe it'll be better to write under later tech debts, as there are lots of other pending stories